sim/team-validator.ts

```diff
@@ -927,8 +927,8 @@ export class TeamValidator {
 
 		let totalEV = 0;
 		for (const stat in set.evs) totalEV += set.evs[stat as 'hp'];
-
-		if (!this.format.debug) {
+		// Not having this affect Nintendo Cup formats because it is annoying to deal with having to lower a base stat by 1 for every Pokemon.
+		if (!this.format.debug && !this.format.cupLevelLimit) {
```

Comment:
there are more intuitive ways to circumvent this that numerous metagames would prefer, but that's for another time.

Reply:
What's going on with Nintendo Cup formats? If I'm understanding this correctly, this seems like the correct way to do this to me.

Reply:
I meant just a generalized `Ignore EV minimum` property or something